File: Java/cycle1/prime.java
package cycle1;
public class prime{
    public static void main(String args[])
    {
        int i,num=3,found=0;
        if (num==0 || num==1)
            System.out.println(num+" is not a prime number");
        else
        {
            for (i=2;i<(num/2);i++)
                if (num%i==0)
                {
                    System.out.println(num+" is not a prime number");
                    found=1;
                    break;
                }
            if (found==0)
                System.out.println(num+" is a prime number");
        }
    }
}

File: C/cycle1/prime.c
#include <stdio.h>
#include <math.h>
void main()
{
    int n,count=0,fact,num=2,sqnum,r,pre;
    printf("Enter the number of primes required: ");
    scanf("%d",&n);
    printf("The first %d prime numbers are ",n);
    while (count<n)
    {
        pre=0;
        fact=2;
        sqnum=sqrt(num);
        while (fact<=sqnum)
        {
            r=num%fact;
            if (r==0)
            {
                pre=1;
                break;
            }
            else
                fact++;
        }
        if (!pre)
        {
            printf("%d ",num);
            count++;
        }
        num++;
    }
    printf("\n");
}